platform/openide.nodes/src/org/openide/nodes/BeanChildren.java

```diff
@@ -48,8 +48,8 @@
      * and child, resp.
      * See #7925.
      */
-    private static final java.util.Map<Node,Reference[]> nodes2Beans = 
-            new WeakHashMap<Node,Reference[]>(); // Map<Node,Reference<Object>[2]>
+    private static final java.util.Map<Node, Reference<?>[]> nodes2Beans
```

Comment:
Formatting style change - is this intentional?

Reply:
I've had NetBeans format the changed lines.
So yes, Intentional.